tests/system-test/3-connectors/c#/TDengineDriver/TaosBind.cs

```diff
@@ -321,7 +323,10 @@ public static void FreeTaosBind(TAOS_BIND[] binds)
         {
             foreach (TAOS_BIND bind in binds)
             {
-                Marshal.FreeHGlobal(bind.buffer);
+                if (bind.is_string)
```

Comment:
since struct need to keep as same as taosc's construct. 
you can change to  use bind.buffer_type = nchar,binary ,json, to adjust if it is a buffer for string.